docs/source/reference/router/telemetry/instrumentation/standard-instruments.mdx

```diff
@@ -10,23 +10,11 @@ GraphOS Router and Apollo Router Core provide a set of non-configurable metric i
 
 These instruments can be consumed by configuring a [metrics exporter](/router/configuration/telemetry/exporters/metrics/overview).
 
-### HTTP
-
-- `apollo_router_http_request_duration_seconds_bucket` - HTTP router request duration
-- `apollo_router_http_request_duration_seconds_bucket` - HTTP subgraph request duration, attributes:
-  - `subgraph`: (Optional) The subgraph being queried
-- `apollo_router_http_requests_total` - Total number of HTTP requests by HTTP status
-
 ### GraphQL
 
 - `apollo.router.graphql_error` - counts GraphQL errors in responses, attributes:
   - `code`: error code
 
-### Session
-
-- `apollo_router_session_count_total` - Number of currently connected clients
```

Comment:
Is this one completely gone in 2.0? is there still some way to track the amount of open connections (not requests)?

Reply:
Yes, I'll document it